bring-your-own-datacenter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: bring-your-own-datacenter
 type: application
-version: 1.3.71
+version: 1.3.72

```

Comment:
This is just the chart version right? Is this aligned with the middleware release?